sycl/test/basic_tests/interop-hip.cpp

```diff
@@ -1,7 +1,6 @@
 // REQUIRES: hip_be
 // RUN: %clangxx %fsycl-host-only -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s
 // RUN: %clangxx %fsycl-host-only -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note -D__SYCL_INTERNAL_API %s
-// expected-no-diagnostics
```

Comment:
Just wondering since we're removing this will the test still fail if there's another diagnostic in addition to the one we're matching for?

Reply:
Yes tests will fail for any unmatched diagnostics (or if the diagnostic is in an unexpected place)